packages/jsii-dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/ComplianceTests.cs

```diff
@@ -260,21 +260,21 @@ public void GetAndSetEnumValues()
 
             calc.Add(9);
             calc.Pow(3);
-            Assert.Equal(CompositionStringStyle.Normal, calc.StringStyle);
-            calc.StringStyle = CompositionStringStyle.Decorated;
-            Assert.Equal(CompositionStringStyle.Decorated, calc.StringStyle);
+            Assert.Equal(CompositionStringStyle.NORMAL, calc.StringStyle);
+            calc.StringStyle = CompositionStringStyle.DECORATED;
+            Assert.Equal(CompositionStringStyle.DECORATED, calc.StringStyle);
             Assert.Equal("<<[[{{(((1 * (0 + 9)) * (0 + 9)) * (0 + 9))}}]]>>", calc.ToString());
         }
 
         [Fact(DisplayName = Prefix + nameof(EnumFromScopedModule))]
         public void UseEnumFromScopedModule()
         {
             ReferenceEnumFromScopedPackage obj = new ReferenceEnumFromScopedPackage();
-            Assert.Equal(EnumFromScopedModule.Value2, obj.Foo);
-            obj.Foo = EnumFromScopedModule.Value1;
-            Assert.Equal(EnumFromScopedModule.Value1, obj.LoadFoo());
-            obj.SaveFoo(EnumFromScopedModule.Value2);
-            Assert.Equal(EnumFromScopedModule.Value2, obj.Foo);
+            Assert.Equal(EnumFromScopedModule.VALUE2, obj.Foo);
```

Comment:
wait... what's the convention for .NET? Do we want .NET to also use all-caps?
Same question for Python.... @garnaat is it idiomatic to use all caps for enum values?

Reply:
- .NET seems to prefer PascalCase.
- Python looks like ALL_CAPS are good.

So basically, the .NET code generator needs to covert the case for .NET.

Reply:
that's a good question. I thought we wanted to avoid dropping into the language specific mangling... let me check.

Reply:
I think Pascal casing for both names and values are the idiomatic thing for .NET

Reply:
No. We wanted to normalize the typescript/jsii source to CAPITALS but then, jsii is all about language-specific idiomacy.

Reply:
Sounds good. I'll make the changes to the generation of .NET so we can change enum values into PascalCase.

Reply:
we decided we would make the changes for .NET generation to use enum members in `PascalCase` outside of this PR. 

Should I create an issue for it before resolving this conversation?

Reply:
Sure, "right click, create issue"